Tests/KonductorIntegrationTests/resources/locationHint.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We can keep this for now, but we may not necessarily need to keep the asserts in physical json files. If they are short enough per test, we can use inline string jsons. Otherwise we need to come up with a consistent test case name + file name convention